packages/oas-utils/src/entities/spec/parameters.ts

```diff
@@ -30,6 +30,8 @@ export const oasParameterSchema = z.object({
   content: z.unknown().optional(),
   /** Defaulted according to @url https://spec.openapis.org/oas/v3.1.0#parameter-object */
   style: parameterStyleSchema.optional(),
+  example: z.unknown().optional(),
+  examples: z.array(z.unknown()).optional(),
```

Comment:
I believe this is a record keyed by example name and not an array. 

![image](https://github.com/user-attachments/assets/5751653a-6b97-4398-b045-e460259654f9)

Can you also add a unit test for this schema? We should really start adding them for all schemas.

Reply:
oh doo catch, i added this one too quickly